### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by mgastonportillo** [mgastonportillo/nvchad-config#5] [Announcements]
## 👋 Welcome, stargazer!
I plan to use this space primarily to receive feedback, but feel free to also:

  * Ask any questions you have
  * Share your ideas
  * Discuss related topics

Please be respectful and open-minded while interacting with others.
  
  > [!NOTE]
  > If you want to report an issue, please go [here](https://github.com/mgastonportillo/nvchad-config/issues/new/choose).

**Comment by mrpmohiburrahman** [comment]
Hi @mgastonportillo , big fan of your nvchad-config.

I’m facing an issue mentioned in https://github.com/NvChad/NvChad/issues/2950

If you or anyone else has any insights or solutions, it would be greatly appreciated.

But may be it's unrelated.

Anyway, Keep up the great work!

**Reply by mgastonportillo** [reply]
Hey there!

I'm honoured that you like my config. 

I've checked the issue you linked and the feature request by siduck. I'm afraid they're right; it is a feature that needs to be implemented as part of nvim-tree. I can look into it tomorrow, but I can't promise to find a solution.
